File: Problem-007/solution.py
def is_prime(num):
    if num < 2:
        return False
    if num == 2:
        return True
    if num % 2 == 0:
        return False

    i = 3
    while i * i <= num:
        if num % i == 0:
            return False
        i += 2

    return True


def nth_prime(n):
    count = 0
    number = 1

    while count < n:
        number += 1
        if is_prime(number):
            count += 1

    return number

File: Problem-007/solution.js
function isPrime(n) {
  if ( n<2) return false;
  if (n === 2 ) return true;
  if (n%2 ===0) return false;

  for (let i = 3; i*i<= n ; i+=2){
    if (n%i ===0){
      return false;
    }
  }

  return true;
}

function nthPrime(n) {
  let count = 0;
  let number = 1;
  while(count <n){
    number++;
    if(isPrime(number)){
      count++;

    }
  }
  return number;
}
